test/unit/app/messaging/process-application-payment-request.test.js

```diff
@@ -30,7 +31,7 @@ describe(('Process application payment request'), () => {
     , receiver)
 
     expect(savePaymentRequest).toHaveBeenCalledTimes(1)
-    expect(receiver.completeMessage).toHaveBeenCalledTimes(1)
+    expect(receiver.completeMessage).toHaveBeenCalledTimes(0)
```

Comment:
Shouldn't this still be being called? It is still there line 44 of process-payment-response

Reply:
Thanks. Fixed.